apps/src/codebridge/FileBrowser/hooks/usePrompts.ts

```diff
@@ -12,11 +13,17 @@ export const usePrompts = () => {
 
   const {project, newFolder} = useCodebridgeContext();
 
+  const sendCodebridgeAnalyticsEvent = useCallback(
+    (event: string) => globalSendCodebridgeAnalyticsEvent(event, appName),
+    [appName]
+  );
+
```

Comment:
this is mostly for purposes of mocking tests, but we can now hand in a new analytics function to the various prompts.